lua/entities/gmod_wire_expression2/core/debug.lua

```diff
@@ -416,3 +416,36 @@ e2function void entity:printColorDriver(array arr)
 
 	printColorArray(self.entity, driver, false, arr)
 end
+
+util.AddNetworkString( "wire_expression2_set_clipboard_text" )
+local clipboard_character_limit = CreateConVar("wire_expression2_clipboard_character_limit", 512, FCVAR_ARCHIVE, "Maximum character that can be copied into a players clipboard", 0, 65532)
+local clipboard_cooldown = CreateConVar("wire_expression2_clipboard_cooldown", 1, FCVAR_ARCHIVE, "Cooldown for setClipboardText in seconds", 0, nil)
+
+
+-- TODO: Make an E2Lib.RegisterChipTable function that is essentially WireLib.RegisterPlayerTable, but handles chips.
```

Comment:
I believe `self.data` is meant to be used like this, but it doesn't really matter that much.

Reply:
Nobody's used self.data for ages and I'm against using it anyway since it might create issues with dupes.